### ISSUE TITLE
ENH: move testing to Github Actions

### ISSUE THREAD

**Issue by leloup314** [SiLab-Bonn/beam_telescope_analysis#23]
Move testing to Github actions.

**Comment by YannickDieter** [comment]
Already done here #19  ;)

**Comment by leloup314** [comment]
Amazing

**Comment by leloup314** [comment]
So we close this issue, once #19 is merged ;)
